engine/access/state_stream/backend/backend.go

```diff
@@ -212,3 +215,9 @@ func (b *StateStreamBackend) getStartHeight(startBlockID flow.Identifier, startH
 func (b *StateStreamBackend) setHighestHeight(height uint64) bool {
 	return b.highestHeight.Set(height)
 }
+
+// GetRegisterValues returns the register values for the given register IDs at the given block height.
+// TODO: find a better home for this method.
+func (b *StateStreamBackend) GetRegisterValues(ids flow.RegisterIDs, height uint64) ([]flow.RegisterValue, error) {
+	return b.registers.RegisterValues(ids, height)
```

Comment:
please add some error handling here. this should check for the sentinels and return appropriate status code versions. e.g. `storage.ErrNotFound` => `codes.NotFound`, `storage.ErrHeightNotIndexed` => `codes.OutOfRange`, etc